src/editor/editor_main.cpp

```diff
@@ -43,8 +44,13 @@ std::string initialize_addon()
 	}
 
 	if(addon_id == "///newaddon///") {
+		std::string& addon_id_new = addon_id;
 		std::int64_t current_millis = std::chrono::duration_cast<std::chrono::milliseconds>(std::chrono::system_clock::now().time_since_epoch()).count();
 		addon_id = "MyAwesomeAddon-"+std::to_string(current_millis);
```

Comment:
@stevecotton Perhaps addon_id also needs to be a translatable string?

Reply:
Why would the addon id be translatable?

Reply:
Looks like I misunderstood something. I thought [this comment](https://github.com/wesnoth/wesnoth/pull/8066#discussion_r1423573578) considered both schedule ID and name to be translatable, but actually it was just the schedule name that has to be translatable. Please ignore this comment.